p2p_wallet/Resources/Base.lproj/Localizable.strings

```diff
@@ -907,3 +907,8 @@ another";
 "We’ve broke something really big.\nLet’s wait together, finnaly the app will be repaired.\n\nIf you will write us, use error code %@" = "We’ve broke something really big.\nLet’s wait together, finnaly the app will be repaired.\n\nIf you will write us, use error code %@";
 "We can’t SMS you" = "We can’t SMS you";
 "Something wrong with phone number %@ or settings. If you will write us, use error code #10464" = "Something wrong with phone number %@ or settings. If you will write us, use error code #10464";
+"You’ve find a seldon page 🦄 It’s like a unicorn, but it’s a crush. We’re already fixing it. If you will write us, use error code %@";
```

Comment:
Why there is no value for this key?

Reply:
Have you used LocalizationHelper to add keys?

Reply:
fixed. but there are more missed values, I will add them consequently later